src/ComScoreStreamingAnalytics.ts

```diff
@@ -82,10 +90,43 @@ export class ComScoreStreamingAnalytics {
     this.player.on(this.player.exports.PlayerEvent.StallStarted, this.stallStarted);
     this.player.on(this.player.exports.PlayerEvent.StallEnded, this.stallEnded);
     this.player.on(this.player.exports.PlayerEvent.AdBreakStarted, this.adBreakStarted);
+    this.player.on(this.player.exports.PlayerEvent.TimeChanged, this.timeChanged);
+  }
+
+  private blured = () => {
+    if (this.player.ads.isLinearAdActive()) {
+      this.stopComScoreTrackingBackgroundAd();
+      console.log('BLURED');
+    }
+  }
+
+  private timeChanged = (event: PlaybackEvent) => {
+    if ( ( this.playerTransitioningState === PlayerTransitionState.TransitioningToVideo ) && ( this.comScoreState !== ComScoreState.Video ) ) {
```

Comment:
Why not calling the existing `transitionToAd` /  `transitionToVideo` methods and keep the code there?